hub-templates/base-hub/templates/nfs-share-creator.yaml

```diff
@@ -31,10 +31,12 @@ spec:
           command:
             - /bin/sh
             - -c
-            - "mkdir -p ${NFS_SHARE_NAME} && chown 1000:1000 ${NFS_SHARE_NAME}"
+            # ${NFS_SHARE_NAME} starts with `/`
+            - "mkdir -p /mnt${NFS_SHARE_NAME} && chown 1000:1000 /mnt${NFS_SHARE_NAME} && mount && ls -la /mnt/ && sleep 60h"
```

Comment:
Why the additional `&& mount && ls -la /mnt/ && sleep 60h` stuff?

Reply:
Ah, the `sleep 60h` was for debugging and i'll take it out. The rest is for debugging - it took me forever to figure this out :D

Reply:
I supposed it was debugging, this is why I asked about it :wink:

Reply:
Fixed!